lib/internal/util.js

```diff
@@ -278,3 +278,14 @@ promisify.custom = kCustomPromisifiedSymbol;
 
 exports.promisify = promisify;
 exports.customPromisifyArgs = kCustomPromisifyArgsSymbol;
+
+exports.intrinsic = {
+  FunctionApply: Function.prototype.apply,
```

Comment:
[Reflect.apply](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/apply) is the correct answer to this.